scripts/checkNvm.sh

```diff
@@ -1,11 +1,21 @@
 #!/bin/bash
 set -e 
```

Comment:
The use of set -e in this file is a bit tricky IMO, can we just get rid of it if we don't to use it?